tests/Unit/inc/classes/subscriber/CDN/RocketCDN/AdminPageSubscriber/displayRocketcdnStatus.php

```diff
@@ -3,137 +3,167 @@
+//			                 ]
+//		                 );
+//
+//		$this->page->shouldReceive( 'generate' )
+//		           ->once()
+//		           ->with(
+//			           'dashboard-status',
+//			           [
+//				           'is_live_site'    => false,
+//				           'container_class' => ' wpr-flex--egal',
+//				           'label'           => '',
+//				           'status_class'    => ' wpr-isInvalid',
+//				           'status_text'     => 'No Subscription',
+//				           'is_active'       => false,
+//			           ]
+//		           )
+//		           ->andReturn( '' );
+//
+//		$this->expectOutputString( '' );
+//		$this->page->display_rocketcdn_status();
+//	}
+//
+//	public function testShouldOutputNoSubscriptionWhenInactive() {
+//		Functions\when( 'rocket_is_live_site' )->justReturn( true );
+//		Functions\expect( 'get_option' )->never();
+//		Functions\expect( 'date_i18n' )->never();
+//
+//		$this->api_client->shouldReceive( 'get_subscription_data' )
+//		                 ->once()
+//		                 ->willReturn(
+//			                 [
+//				                 'is_active'                     => false,
+//				                 'subscription_status'           => 'cancelled',
+//				                 'subscription_next_date_update' => '2020-01-01',
+//			                 ]
+//		                 );
+//
+//		$expected = <<<HTML
+//<div class="wpr-optionHeader">
+//	<h3 class="wpr-title2">RocketCDN</h3>
+//</div>
+//<div class="wpr-field wpr-field-account">
+//	<div class="wpr-flex wpr-flex--egal">
+//		<div>
+//			<span class="wpr-title3"></span>
+//			<span class="wpr-infoAccount wpr-isInvalid">No Subscription</span>
+//		</div>
+//		<div>
+//			<a href="#page_cdn" class="wpr-button">Get RocketCDN</a>
+//		</div>
+//	</div>
+//</div>
+//HTML;
+//
+//		$this->assertSame( $this->format_the_html( $expected ), $this->getActualHtml() );
+//	}
+//
+//	public function testShouldOutputSubscriptionDataWhenActive() {
+//		Functions\when( 'rocket_is_live_site' )->justReturn( true );
+//		Functions\when( 'get_option' )->justReturn( 'Y-m-d' );
+//		Functions\when( 'date_i18n' )->justReturn( '2020-01-01' );
+//
+//		$this->api_client->method( 'get_subscription_data' )
+//		                 ->willReturn(
+//			                 [
+//				                 'is_active'                     => true,
+//				                 'subscription_status'           => 'running',
+//				                 'subscription_next_date_update' => '2020-01-01',
+//			                 ]
+//		                 );
+//
+//		$expected = <<<HTML
+//<div class="wpr-optionHeader">
+//	<h3 class="wpr-title2">RocketCDN</h3>
+//</div>
+//<div class="wpr-field wpr-field-account">
+//	<div class="wpr-flex">
+//		<div>
+//			<span class="wpr-title3">Next Billing Date</span>
+//			<span class="wpr-infoAccount wpr-isValid">2020-01-01</span>
+//		</div>
+//	</div>
+//</div>
+//HTML;
+//
+//		$this->assertSame( $this->format_the_html( $expected ), $this->getActualHtml() );
+//	}
```

Comment:
should this be preserved?

Reply:
Whoopsie.... Fixed here https://github.com/wp-media/wp-rocket/pull/2470/commits/8209a02ecb018d7fd1a1b6d5688cbd1fd3efd88a.